src/__tests__/utils.js

```diff
@@ -62,6 +62,16 @@ describe('_.copyAndTruncateStrings', () => {
 
         expect(given.subject).toEqual({ key: 'vaaaa', values: ['fooob', undefined], __deepCircularCopyInProgress__: 1 })
     })
+
+    it('should check copy-in-progress correctly', () => {
+        given('target', () => {
+            const base = Object.create({ [COPY_IN_PROGRESS_ATTRIBUTE]: undefined })
+            const object = Object.create(base)
+            return object
+        })
+
+        expect(given.target).not.toEqual(void 0)
```

Comment:
This test does not seem to test neither deepCircularCopy or copyAndTruncateStrings in it's current form.

Did you intend something like:
```
expect(given.subject).toEqual({})
```
?

Reply:
Oops, my bad. 🤣